Delivered-To: koen@dominion.thruhere.net
Received: by 10.194.158.231 with SMTP id wx7csp352612wjb;
        Tue, 27 Nov 2012 06:45:37 -0800 (PST)
Received: by 10.66.79.168 with SMTP id k8mr43446430pax.12.1354027536981;
        Tue, 27 Nov 2012 06:45:36 -0800 (PST)
Return-Path: <linux-omap-owner@vger.kernel.org>
Received: from vger.kernel.org (vger.kernel.org. [209.132.180.67])
        by mx.google.com with ESMTP id d7si2355402paw.327.2012.11.27.06.45.36;
        Tue, 27 Nov 2012 06:45:36 -0800 (PST)
Received-SPF: pass (google.com: best guess record for domain of linux-omap-owner@vger.kernel.org designates 209.132.180.67 as permitted sender) client-ip=209.132.180.67;
Authentication-Results: mx.google.com; spf=pass (google.com: best guess record for domain of linux-omap-owner@vger.kernel.org designates 209.132.180.67 as permitted sender) smtp.mail=linux-omap-owner@vger.kernel.org
Received: (majordomo@vger.kernel.org) by vger.kernel.org via listexpand
	id S1753286Ab2K0Ope (ORCPT <rfc822;koen@dominion.thruhere.net>);
	Tue, 27 Nov 2012 09:45:34 -0500
Received: from bear.ext.ti.com ([192.94.94.41]:36685 "EHLO bear.ext.ti.com"
	rhost-flags-OK-OK-OK-OK) by vger.kernel.org with ESMTP
	id S1751086Ab2K0Opd (ORCPT <rfc822;linux-omap@vger.kernel.org>);
	Tue, 27 Nov 2012 09:45:33 -0500
Received: from dbdp20.itg.ti.com ([172.24.170.38])
	by bear.ext.ti.com (8.13.7/8.13.7) with ESMTP id qAREjUY8011241;
	Tue, 27 Nov 2012 08:45:31 -0600
Received: from DBDE70.ent.ti.com (localhost [127.0.0.1])
	by dbdp20.itg.ti.com (8.13.8/8.13.8) with ESMTP id qAREjSXV010652;
	Tue, 27 Nov 2012 20:15:29 +0530 (IST)
Received: from dbdp32.itg.ti.com (172.24.170.251) by dbde70.ent.ti.com
 (172.24.170.148) with Microsoft SMTP Server id 14.1.323.3; Tue, 27 Nov 2012
 20:15:27 +0530
Received: from psplinux063.india.ti.com (dbdp20.itg.ti.com [172.24.170.38])	by
 dbdp32.itg.ti.com (8.13.8/8.13.8) with ESMTP id qAREjPZf025374;	Tue, 27 Nov
 2012 20:15:25 +0530
From:	Afzal Mohammed <afzal@ti.com>
To:	Felipe Balbi <balbi@ti.com>
CC:	Greg Kroah-Hartman <gregkh@linuxfoundation.org>,
	Tony Lindgren <tony@atomide.com>, <linux-usb@vger.kernel.org>,
	<linux-omap@vger.kernel.org>, Afzal Mohammed <afzal@ti.com>
Subject: [PATCH] usb: musb: dsps: header movement build error fix
Date:	Tue, 27 Nov 2012 20:15:22 +0530
Message-ID: <1354027522-32485-1-git-send-email-afzal@ti.com>
X-Mailer: git-send-email 1.7.12
MIME-Version: 1.0
Content-Type: text/plain
Sender:	linux-omap-owner@vger.kernel.org
Precedence: bulk
List-ID: <linux-omap.vger.kernel.org>
X-Mailing-List:	linux-omap@vger.kernel.org

"54db6ee ARM: OMAP2+: Introduce local usb.h" moved control module bit
definitions from plat/usb.h (which dsps glue was using) to a local
header in mach-omap2. And in parallel,
"c68bb4c usb: musb: dsps: control module handling (quirk)" added
control module handling capability to dsps glue driver that used
those control module bit definitions.

Integration of above two changes would cause build error in musb dsps
glue driver (they go through different trees upstream) as is seen now
in linux-next. Fix it by adding necessary definitions in dsps glue
driver.

Signed-off-by: Afzal Mohammed <afzal@ti.com>
---

Hi Felipe,

This applies on top of your musb branch, please help this reach
mainline along with other musb dsps changes for coming merge window
so that they would not cause build error and so that we can have a
working usb in mainline for am335x (beaglebone) at the earliest.

Regards
Afzal

 drivers/usb/musb/musb_dsps.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/drivers/usb/musb/musb_dsps.c b/drivers/usb/musb/musb_dsps.c
index e6f2ae8..f7d764d 100644
--- a/drivers/usb/musb/musb_dsps.c
+++ b/drivers/usb/musb/musb_dsps.c
@@ -134,6 +134,11 @@ static const resource_size_t dsps_control_module_phys[] = {
 	DSPS_AM33XX_CONTROL_MODULE_PHYS_1,
 };
 
+#define USBPHY_CM_PWRDN		(1 << 0)
+#define USBPHY_OTG_PWRDN	(1 << 1)
+#define USBPHY_OTGVDET_EN	(1 << 19)
+#define USBPHY_OTGSESSEND_EN	(1 << 20)
+
 /**
  * musb_dsps_phy_control - phy on/off
  * @glue: struct dsps_glue *
-- 
1.7.12

--
To unsubscribe from this list: send the line "unsubscribe linux-omap" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
